dipy/testing/decorators.py

```diff
@@ -52,9 +53,15 @@ def doctest_skip_parser(func):
 # xvfbwrapper) conditioned on an environment variable (that we set in
 # .travis.yml for these cases):
 use_xvfb = os.environ.get('TEST_WITH_XVFB', False)
+if use_xvfb == 'true':
```

Comment:
Do it! Please.

Ugh - I have spent all day on these bots, and still no joy. If this doesn't
work, I suggest that we go to skipping these tests on the bots.

On Wed, Nov 25, 2015 at 12:09 PM, Eleftherios Garyfallidis <
notifications@github.com> wrote:

> In dipy/testing/decorators.py
> <https://github.com/nipy/dipy/pull/782#discussion_r45913835>:
>
> > @@ -52,9 +53,15 @@ def doctest_skip_parser(func):
> >  # xvfbwrapper) conditioned on an environment variable (that we set in
> >  # .travis.yml for these cases):
> >  use_xvfb = os.environ.get('TEST_WITH_XVFB', False)
> > +if use_xvfb == 'true':
>
> Okay, if you agree I can go ahead and merge your PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/782/files#r45913835>.
>